swarm/src/handler/select.rs

```diff
@@ -208,8 +208,8 @@ where
         }
     }
 
-    fn connection_keep_alive(&self) -> KeepAlive {
-        cmp::max(
+    fn connection_keep_alive(&self) -> bool {
+        cmp::min(
```

Comment:
I think this should be `max` but overall, it is probably easier to understand if we write it as `|`.

Reply:
I misunderstand `Ord ` impl on `KeepAlive` , reverted it back 69a4a8b already